src/playground/tabs/SettingsTab.tsx

```diff
@@ -346,6 +360,38 @@ export default function SettingsTab({
 	);
 }
 
+function LanguageView({
+	language,
+	setLanguage,
+}: { language: Language; setLanguage: (language: Language) => void }) {
+	return (
+		<section>
+			<div className="field-row">
+				<label htmlFor="language">Language</label>
+				<select
+					id="language"
+					name="language"
+					value={language ?? Language.TSX}
```

Comment:
Not a blocker, I wonder if there's a way to enforce a compiler failure in case we add new languages. I am concerned that once we add a new variant to `Language`, we forget to update this

Reply:
I think adding a new component that maps a enum to a string text will help in this case. It will not be a straightforward representation for HTML, but missing case will never happen.